docs/platforms-release-process.md

```diff
@@ -357,25 +363,31 @@ Create a JIRA issue for it, and mark it as a blocker.
 
 All good? Have another look at the changes:
 
-    coho repo-status -r android -b master -b 5.0.x
+    coho repo-status -r android -b master -b 7.1.x
 
 If changes look right:
 
     coho repo-push -r android -b master -b 5.0.x
 
-This pushes the commits in both `master` and `5.0.x` (the release branch) to the remote.
+This pushes the commits in both `master` and `7.1.x` (the release branch) to the remote.
 
 ### Tag and push tag
 
 Before you tag, run this command:
 
-    coho tag-platform-release --version 5.0.0 -r android --pretend
+    coho tag-platform-release --version 7.1.0 -r android --pretend
     
 Seems okay? Then execute it by running:
 
-    coho tag-platform-release --version 5.0.0 -r android
+    coho tag-platform-release --version 7.1.0 -r android
+
+This command also tags `cordova-js` with `android-7.1.0` and pushes it.
+
+To tag without automatic push:
```

Comment:
Why would one want to use that instead of automatic push?

Reply:
I personally do not like automatic push. I would rather review my most recent changes before pushing to GitHub.